src/main/kotlin/com/openlattice/auditing/AuditingProfiles.kt

```diff
@@ -0,0 +1,13 @@
+package com.openlattice.auditing
+
+class AuditingProfiles {
+
+    companion object {
+
+        @JvmField
+        val LOCAL_AUDITING_PROFILE = "auditlocal"
+
+        @JvmField
+        val LOCAL_AWS_AUDITING_PROFILE = "auditlocalaws"
```

Comment:
Am confused about what this would be auditing; AWS? or local machine?

Reply:
it's for running locally while auditing to the aws test bucket (instead of to the local machine) -- I was following the `medialocal`, `medialocalaws` pattern but can change it if it's confusing!

Reply:
I think it's fine for now but I find the `*local` vs `*localaws` pattern to be unintuitive. I think it makes more sense to call them `*local` and `*aws` for running against a `local` target and an `aws` target respectively